src/common_interface/solve.jl

```diff
@@ -1432,7 +1432,6 @@ function DiffEqBase.solve!(integrator::AbstractSundialsIntegrator; early_free =
                     integrator.opts.save_idxs)
         push!(integrator.sol.t, integrator.t)
         if integrator.opts.dense
-            integrator(integrator.u, integrator.t, Val{1})
```

Comment:
I don't think this is correct with CVODE. With IDA it will naturally use and update `du`, but with CVODE `du` is not used in the computation at all, so this might need to have an `if CVODE || ARKODE`.